unblob/testing.py

```diff
@@ -80,5 +81,11 @@ def check_output_is_the_same(reference_dir: Path, extract_dir: Path):
 
 def check_result(reports: ProcessResult):
     __tracebackhide__ = True
-
-    assert reports.errors == [], "Unexpected error reports"
+    # filter out error reports about truncated integration test files
+    errors = [
+        error
+        for error in reports.errors
+        if not isinstance(error, ExtractCommandFailedReport)
+        or error.stderr != b"\nERRORS:\nUnexpected end of archive\n\n"
```

Comment:
Did you meant to skip every instance of `ExtractCommandFailedReport` regardless of the error message? Maybe I misunderstood something, but didn't we agree only ignoring this one very specific message and nothing else?

```suggestion
        if not (
            isinstance(error, ExtractCommandFailedReport)
            and error.stderr == b"\nERRORS:\nUnexpected end of archive\n\n"
        )
```
I'm fine skipping all of these, just wanted to clarify if it was the intention.

Reply:
Indeed. Fixed.

Reply:
FYI, `(not A) or (not B)` is the same as `not (A and B)`, i.e. it was not a fix: nothing changed.